MSAL/src/instance/MSALAuthority.m

```diff
@@ -163,8 +163,12 @@ + (void)resolveEndpointsForAuthority:(NSURL *)unvalidatedAuthority
     
     if (authorityInCache)
     {
-        completionBlock(authorityInCache, nil);
-        return;
+        if (!validate ||
```

Comment:
I realized it wasn't actually being taken from there (or from anywhere else). This should fix it: #149